src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -284,23 +289,28 @@ class boss_archimonde : public CreatureScript
 
             Enraged = false;
             HasProtected = false;
+            summons.DespawnAll();
+            WorldtreeTragetGUID=instance->GetGuidData(DATA_CHANNEL_TARGET);
+            if (Creature* WorldtreeTraget = ObjectAccessor::GetCreature(*me, WorldtreeTragetGUID)){
+                DoCast(WorldtreeTraget, SPELL_DRAIN_WORLD_TREE);
```

Comment:
Initialize() method is supposed to only initialize C++ class member fields. It's not supposed to cast spells or unsummon mobs.
Please move this part to Reset() or wherever it's supposed to be

Reply:
Now I wonder, is this spell actually even cast ingame ? did you check if it works ?

Reply:
Thank you for reply.The spell only has a visual effect,found in sniff and video.